src/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <script
+      src="https://kit.fontawesome.com/08b4cd9ea0.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Home Page</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <div class="logo-container">
+          <h3 id="logo">My Logo</h3>
+        </div>
+        <label for="check" class="hamberger-btn">
+          <i id="ham" class="fas fa-bars"></i>
+        </label>
+      </nav>
+    </header>
+
+    <section class="section">
+      <div class="container">
+        <div class="flex">
+          <h1 id="my_name">
+            Hey There. <br />
+            I'm Zola.
+          </h1>
+          <h4 id="title">I'm a software Developer</h4>
+          <p id="describe_me">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hesitate to contact me.
+          </p>
```

Comment:
- Kindly make sure you use the appropriate font family to make your design stick to the figma design as required in the project :+1: 
-Kinndly  For the title `<h1 id="my_name"> Hey There.  <br />I'm Zola. </h1>` reduce the font size so that it will make it in two lines instead of three as shows in the figma design :+1: 

## Your design 

![image](https://user-images.githubusercontent.com/95297251/235137007-49c0b3b1-455e-4cfb-bf20-2c8ab0606cb4.png)

## Figma design 
![image](https://user-images.githubusercontent.com/95297251/235136859-7132b4d5-48ea-491d-8014-768846a7cfb9.png)